src/site/_data/authorsData.json

```diff
@@ -1093,5 +1093,15 @@
     "twitter": "brendankenny",
     "github": "brendankenny",
     "image": "image/MtjnObpuceYe3ijODN3a79WrxLU2/sWOfAs4f2WyQMMF9xt1A.jpg"
+  },
+  "svenmay": {
+    "country": "DE",
+    "github": "Svenmay",
+    "image": "image/dB6B4Sr8kaaT0KZujRBFC303oFR2/PqRYDSerevsQvWa5ixxW.jpg"
```

Comment:
Per [web.dev media standards], author headshots should be a 384px square. This one is 600x600.